src/Spatial/Euclidean/CoordinateSystem.cs

```diff
@@ -258,13 +258,44 @@ public static CoordinateSystem Rotation(Angle angle, Vector3D v)
         /// <returns>A rotated coordinate system</returns>
         public static CoordinateSystem Rotation(Angle yaw, Angle pitch, Angle roll)
         {
-            var cs = new CoordinateSystem();
             var yt = Yaw(yaw);
             var pt = Pitch(pitch);
             var rt = Roll(roll);
-            return rt.Transform(pt.Transform(yt.Transform(cs)));
+            return yt.Transform(pt.Transform(rt));
         }
 
+        /// <summary>
+        /// Create a rotated CoordinateSystem in one calculation instead of 3 separate rotations.
```

Comment:
I would add a URL link to the wiki page section which documents this calculation